src/renderer/state.ts

```diff
@@ -725,25 +717,25 @@ export class AppState {
     // mount to ensure that we can load content into the editors as soon as they're
     // declared visible.
 
-    await waitForEditorsToMount(visible);
+    await waitForEditorsToMount(visible.filter(isKnownFile));
```

Comment:
why are we differentiating between custom/default editors here?

Reply:
good catch. It shouldn't be. Fixed fce9f174e1bdce94ca8fa4ba569bd6d7a730ee2c